qoe-metrics-dashboard/src/app/models/types/responses/metrics-details-report.interface.ts

```diff
@@ -0,0 +1,37 @@
+import { QoeMetric } from '../qoe-report.type';
+
+/**
+ * The response type for the backend endpoint for metrics report detail
+ */
+export type TMetricsDetailsReportResponse = TMetricsDetailReport[];
+
+/**
+ * The interface for a single metrics report detail
+ */
+export interface TMetricsDetailReport {
+    ReceptionReport: IReceptionReport;
+}
+
+/**
+ * Template for a reception report
+ */
+export interface IReceptionReport {
+    clientID: string;
+    contentURI: string;
+    'xsi:schemaLocation': string;
```

Comment:
Das sieht nicht so aus als sollte es in einem Interface sein

Reply:
Mag sein, aber das ist sein code nicht meiner und ich würde auch solche unkonventionellen Properties zugänglich mache über ein Interface.